openlibrary/templates/home/welcome.html

```diff
@@ -66,6 +66,15 @@ <h2 class="home-h2">$_("Welcome to Open Library")</h2>
       $ }
       $:render_template('home/onboarding_card', volunteer_card['card_title'], volunteer_card['card_body'], volunteer_card['img_src'], volunteer_card['href'], ol_link_track=volunteer_card['ol_link_track'])
 
+      $ volunteer_action_card = {
+      $    "card_title": _("Volunteer at Open Library"),
+      $    "card_body": _("Volunteer at Open Library"),
+      $    "img_src": "/static/images/onboarding/volunteer.png",
+      $    "href": "/volunteer",
+      $    "ol_link_track": "OnboardingCarouselClick|Volunteer"
+      $ }
+      $:render_template('home/onboarding_card', volunteer_action_card['card_title'], volunteer_action_card['card_body'], volunteer_action_card['img_src'], volunteer_action_card['href'], ol_link_track=volunteer_action_card['ol_link_track'])
```

Comment:
```suggestion
      $:render_template('home/onboarding_card', volunteer_action_card['card_title'], volunteer_action_card['card_body'], volunteer_action_card['img_src'], volunteer_action_card['href'], ol_link_track=volunteer_action_card['ol_link_track'], attribution=volunteer_action_card['attribution'])
```
Include attribution.